sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/android/PackageName.kt

```diff
@@ -30,6 +30,7 @@ internal fun Project.packageName(): String {
 internal fun Project.sqliteVersion(): String? {
   val androidExtension = extensions.getByType(BaseExtension::class.java)
   val minSdk = androidExtension.defaultConfig.minSdk ?: return null
-  if (minSdk >= 30) return "app.cash.sqldelight:sqlite-3-25-dialect:$VERSION"
+  if (minSdk == 30) return "app.cash.sqldelight:sqlite-3-25-dialect:$VERSION"
+  if (minSdk >= 31) return "app.cash.sqldelight:sqlite-3-30-dialect:$VERSION"
```

Comment:
This pattern is a bit safer and easier to read for the future as we add more. 
```suggestion
  if (minSdk >= 31) return "app.cash.sqldelight:sqlite-3-30-dialect:$VERSION"
  if (minSdk >= 30) return "app.cash.sqldelight:sqlite-3-25-dialect:$VERSION"
```